mahta/TS/TS basics/basics.ts

```diff
@@ -0,0 +1,73 @@
+function add(a: number, b: number): number {
```

Comment:
How did you run the ts code?
I don't find the tsconfig.json.
Please configure your TypeScript.

Reply:
you can use https://typestrong.org/ts-node/docs/ to run ts code, also you can use npx tsc [your file]

Reply:
The self "TypeScript" infers the output type.
This means that when two numbers are added together, the Typescript understands that the output is a number and it's unnecessary to explicitly declare what the output is